search/src/search.rs

```diff
@@ -17,6 +42,7 @@ pub struct Search {
 impl Search {
     pub fn new(index: Index, schema: SearchSchema) -> Result<Self> {
         let reader = index
+            .index()
```

Comment:
I think it would be cleaner / more decoupled not to give external access to the tantivy index through our index module.
If the search module "needs" it to build a reader or query parser, we should provide methods on the index that returns those instead.

Reply:
I fully agree. Shall we merge the PR anyway and create a new issue for this?

Reply:
Yes, you're right, it should be separate!